include/fmt/compile.h

```diff
@@ -31,6 +31,22 @@ class compiled_string {};
 template <typename S>
 struct is_compiled_string : std::is_base_of<compiled_string, S> {};
 
+#ifdef __cpp_if_constexpr
+# define FMT_COMPILE_IMPL(s) FMT_STRING_IMPL(s, fmt::detail::compiled_string)
+#else
+namespace fmt_compile_error {
+  template<class T> struct false_type : std::false_type {};
+  
+  template<class T> FMT_CONSTEXPR const char* report_compile_error() {
+    static_assert(
```

Comment:
Seems like broken code-style for me

Reply:
If you mean that both scopes are on separate lines, while arguments are indented and are on separate line? It was done by purpose. Because GCC and clang output full line with the failed static_assert's condition. So I think that it's better to place not so interesting parts of this code aside.

Reply:
IMHO, code style should be either applied for entire code or disabled with special `//clang format off` directive.
Don't think that you as the end user would somehow analyze the code that fail `static_assert` since you already have verbose error with `static_assert` string.